atom/browser/atom_browser_client.cc

```diff
@@ -291,4 +291,26 @@ void AtomBrowserClient::RenderProcessHostDestroyed(
   }
 }
 
+bool AtomBrowserClient::IsHandledURL(const GURL& url) {
+  if (!url.is_valid())
+    return false;
+  // Keep in sync with ProtocolHandlers added by
+  // AtomBrowserContext::GetURLRequestContext().
+  static const char* const kProtocolList[] = {
```

Comment:
I think we should call `URLRequestJobFactory::IsHandledProtocol` because we have some custom protocols registered in `atom::api::Protocol`

Reply:
I think it also need `net::URLRequest::IsHandledProtocol(scheme);` and can remove the http/ws/ftp protocols. See `ProfileIOData::IsHandledProtocol`

Reply:
@darkdh, the atom_browser_client do not have access to URLRequestJobFactory or browsercontext so we cannot use URLRequestJobFactory::IsHandledProtocol. Thats the reason we need to keep them in sync manually with all protocol handlers added by embedder in AtomBrowserContext::CreateURLRequestJobFactory. I have no knowledge on electron/atom api, i request you to add more protocol in the list if the atom protocol api adds one.

@ bridiver, added net::URLRequest::IsHandledProtocol(scheme). 
cannot remove the http/ws/ftp because these protocol handlers are added in AtomBrowserContext::CreateURLRequestJobFactory

Reply:
@darkdh, the atom_browser_client do not have access to URLRequestJobFactory or browsercontext so we cannot use URLRequestJobFactory::IsHandledProtocol. Thats the reason we need to keep them in sync manually with all protocol handlers added by embedder in AtomBrowserContext::CreateURLRequestJobFactory. I have no knowledge on electron/atom api, i request you to add more protocol in the list if the atom protocol api adds one.

@bridiver, added the change for net::URLRequest::IsHandledProtocol(scheme).
cannot remove the http/ws/ftp because these protocol handlers are added in AtomBrowserContext::CreateURLRequestJobFactory

Reply:
@gnanasekar-somanathan http/https/ws/wss are included in `net::URLRequest::IsHandledProtocol`

Reply:
I think we also need `content::kChromeDevToolsScheme`, `extensions::kExtensionScheme`, `url::kAboutScheme` and `content::kChromeUIScheme`

Reply:
should mostly match `ProfileIOData::IsHandledProtocol`, but we don't need the chromeos, android, `dom_distiller::kDomDistillerScheme,` or `chrome::kChromeSearchScheme`

Reply:
also please move to brave_content_browser_client. atom_browser_client is deprecated and will be merged into brave_content_browser_client